src/components/shared/nav/nav.jsx

```diff
@@ -1,5 +1,36 @@
+import Link from 'next/link';
+import EmailListArrow from '../svg/EmailListArrow/EmailListArrow';
+import NoraLogo from '../svg/NoraLogo/NoraLogo';
 import styles from './nav.module.scss';
 
 export default function Nav() {
-  return <nav>Sample nav</nav>;
+  const navLinks = ['Home', 'Paintings', 'Exhibitions', 'About', 'Contact'];
```

Comment:
for each page, lets use an object that has the label and href. This will make it easier to manage in the future if we need to add data to any of these links, for example if we need to add any query params.